syntax = "proto3";

message Credentials {
    string email = 1;
	string password = 2;
}

message Token {
    string token = 1;
    int64 expires_in = 2;
}

message UserDetails {
    string identifier = 1;
    string first_name = 2;
    string last_name = 3;
    string email_address = 4;
    string photo_url = 5;
}